NNLOops.md

```diff
@@ -24,10 +39,31 @@
   </tr>
   <tr>
     <td>For every event, for every Z boson candidate, look for the back-to-back jet and make an histogram of the ratio of their transverse momenta.</td>
+    <td><pre lang="python">
+for e in events:
+    for z in z_candidates:
+        for j in jets:
+            if j.p == -z.p:
+                hist.fill(j.pt/z.pt)
+    </pre></td>
     <td>???</td>
   </tr>
   <tr>
     <td>For every event, for every jet, take all tracks and find the closest track that has pt>1 gev, and then plot the delta-R of that track and jet, the jet's pT, and the tracks pT.</td>
+    <td><pre lang="python">
+for e in events:
+    for j in jets:
+        closest_track = None
+        d = None
+        for t in tracks:
+            if t.pt > 1:
+                if d > (t.p - j.p)**2:
+                    d = (t.p - j.p)**2
+                    closest_track = t
+        hist_jpt.fill(j.pt)
+        hist_tpt.fill(closest_track.pt)
+        hist_dr.fill(sqrt(j.eta**2 + j.phi**2) - sqrt(closest_track.eta**2 + closest_track.phi**2))

```

Comment:
could you add a newline so the table looks more compact? :D

Reply:
added linebreaks and two-space indentations